index.d.ts

```diff
@@ -1,31 +1,29 @@
-export type SkuTypeAndroid = 'INAPP' | 'SUBS'
-export type SkuTypeIOS = 'iap' | 'sub'
-
-export interface SkuTypes {
-  android: { [key: string]: SkuTypeAndroid }
-  ios: { [key: string]: SkuTypeIOS }
+import * as Apple from './apple'
+
+interface Common {
+  title: string
+  description: string
+  price: string
+  currency: string
+  localizedPrice: string
 }
 
-export const SkuTypes: SkuTypes
```

Comment:
I'm removing this because it wasn't actually exported from `index.js`, so trying to use it would just result in `undefined` at runtime.

I'm planning on fixing this by normalizing the `type` property so that it's the same on Android and iOS, expect a PR for that soon™ 😎

Reply:
That's good plan. But breaks compatibility, so I think we'll have to deal with this change separately.

Reply:
> But breaks compatibility

This change wouldn't though? Because there are no `SkuTypes` actually exported from `index.js`?

I still think that we should change it though, shouldn't be a problem to introduce breaking changes since we are in an alpha release?

Reply:
Ah, sorry. I confused. I meant the plan. This change looks good.

But other changes below:
```diff
export interface Product {
-  type: SkuTypeAndroid | SkuTypeIOS;
+ type: 'inapp' | 'iap'
```
This is the plan you said right?

Reply:
> It looks typeof getProducts is not valid for Android until the SkuType is nomalized.

I believe it is, `'inapp'` is for Android and `'iap'` for iOS.

Note that I split up `Product` and `Subscription` since `getProducts` only returns products (I think at some point in history it also included subscriptions on iOS 🤔)

Reply:
I'm not sure about. Alright, If it's case insensitive. :ok_hand: